### ISSUE TITLE
scikit-learn pipeline

### ISSUE THREAD

**Issue by qianmingduowan** [yanqiangmiffy/20newsgroups-text-classification#3]
when you load your dataset, the test set is from the train  , a very stupid mistake
